test/ReactWrapper-spec.jsx

```diff
@@ -3318,11 +3368,28 @@ describeWithDOM('mount', () => {
     });
 
     describeIf(!REACT013, 'stateless components', () => {
-      const SFC = () => (<div />);
+      const SFC = () => (
+        <div className="outer">
+          <div className="inner">
+            <span />
+            <span />
+          </div>
+        </div>
+      );
+
+      it('should return the outermost DOMComponent of the root wrapper', () => {
+        const wrapper = mount(<SFC />);
+        expect(wrapper.getDOMNode()).to.have.property('className', 'outer');
+      });
 
-      it('should throw when wrapping an SFC', () => {
+      it('should return the outermost DOMComponent of the inner div wrapper', () => {
         const wrapper = mount(<SFC />);
-        expect(() => wrapper.getDOMNode()).to.throw(TypeError, 'Method “getDOMNode” cannot be used on functional components.');
+        expect(wrapper.find('.inner').getDOMNode()).to.have.property('className', 'inner');
+      });
+
+      it('should throw when wrapping multiple elements', () => {
+        const wrapper = mount(<SFC />).find('span');
+        expect(() => wrapper.getDOMNode()).to.throw(Error);
```

Comment:
Can we test the error message like we had before? Testing that it throws any error seems brittle.